app/views/help/show.html.erb

```diff
@@ -9,92 +9,28 @@
-              <ul class="list-inside my-2 list-decimal">
-                <li>Click on the three horizontal bars from the top left corner of your screen to expand the menu</li>
-                <li>Select 'Requests' from the menu</li>
-                <li>Add any details for filtering results and select 'Filter'</li>
-                <li>If no filters are necessary, select 'Clear Filters'</li>
-                <li>To see a list of all product totals, select 'Calculate Product Totals'</li>
-                <li>To view these totals in a spreadsheet, select 'Export to CSV'</li>
-              </ul>
-              <li class='text-2xl font-bold list-none'>How do I view my distrutbution calendar?</li>
-              <ul class="list-inside my-2 list-decimal">
-                <li>Select the calendar icon from the page header, OR</li>
-                <li>Click on the three horizontal bars from the top left corner of your screen to expand the menu</li>
-                <li>Select 'Pick Ups &amp; Deliveries' from the menu</li>
-              </ul>
-              <li class='text-2xl font-bold list-none'>How do I add Partner Agencies?</li>
-              <ul class="list-inside my-2 list-decimal">
-                <li>Click on the three horizontal bars from the top left corner of your screen to expand the menu</li>
-                <li>Select 'Partner Agencies' from the menu</li>
-                <li>To add multiple partners at once, select 'Import Partners' and follow the instructions</li>
-                <li>To add a single partner, select 'New Partner Agency', complete the form, and select 'Add Partner Agency'</li>
-                <li>Send an invitation to the new agency by selecting 'Invite' under the Action column</li>
-                <li>To approve a partner, select 'Review Application' from the Action column. If the partner has requested approval, you can select 'Approve Partner'</li>
-              </ul>
-              <li class='text-2xl font-bold list-none'>How do I edit a Partner Agency?</li>
-              <ul class="list-inside my-2 list-decimal">
-                <li>Click on the three horizontal bars from the top left corner of your screen to expand the menu</li>
-                <li>Select 'Partner Agencies' from the menu</li>
-                <li>If a partner has not been approved, select 'Review Application' from the Action column, followed by 'Edit Information'</li>
-              </ul>
-              <li class='text-2xl font-bold list-none'>How do I view or edit my Inventory Items, Kits, Barcode Items, Storage Locations, Inventory Adjustments, or Transfers?</li>
-              <ul class="list-inside my-2 list-decimal">
-                <li>Click on the three horizontal bars from the top left corner of your screen to expand the menu</li>
-                <li>Select 'Inventory' from the menu, followed by the topic you'd like to view</li>
-              </ul>
```

Comment:
We should pre-populate the database with these questions and answers rather than forcing the superuser to enter them all manually.

Reply:
I was wondering about that. I'll work on getting this set up.

Reply:
Do you mean seeding questions for development?

Reply:
We should definitely add some questions to the seeds file. We can _probably_ skip the pre-populating for the prod site -- Angela mentioned wanting to re-word most of the current items so it would probably be wasted effort as they're all mostly going to change.

Reply:
I added some questions to the seeds file. If it needs adjustments, please let me know.